src/Tizen.System/Device/Battery.cs

```diff
@@ -76,11 +76,20 @@ public static class Battery
     {
         private static readonly object s_lock = new object();
         /// <summary>
-        /// Gets the battery charge percentage.
+        /// Gets the current device'sinvalid battery charge percentage as an interger value.
```

Comment:
```suggestion
        /// Gets the current device's invalid battery charge percentage as an interger value.
```

Reply:
I changed that description as you suggested.